libraries/AP_GPS/AP_GPS.h

```diff
@@ -457,6 +473,28 @@ class AP_GPS
 
     // inject data into all backends
     void inject_data_all(const uint8_t *data, uint16_t len);
+
+    // GPS blending and switching
+    Vector2f _NE_pos_offset_m[GPS_MAX_RECEIVERS]; // Filtered North,East position offset from GPS instance to blended solution in _output_state.location (m)
+    float _hgt_offset_cm[GPS_MAX_RECEIVERS]; // Filtered height offset from GPS instance relative to blended solution in _output_state.location (cm)
+    Vector3f _blended_antenna_offset; // blended antenna offset
+    float _blend_weights[GPS_MAX_RECEIVERS]; // blend weight for each GPS. The blend weights must sum to 1.0 across all instances.
+    Location _output_location_corrected; // output location after application of the  filtered offset corrections _NE_pos_offset_m and _hgt_offset_cm
```

Comment:
What is this used for? I can see we do some math with it, but we never read it anywhere that I can find after it's had calculations done on it.